### ISSUE TITLE
syncing network time

### ISSUE THREAD

**Issue by saintscribe69** [thetatoken/guardian-mainnet-guide#94]
ive been trying to set up a node and its taking days to comple syncing the network, its been stuck on 99. something for two days. I did sto-p delete and start again, and the same thing  happened. any7body elese have this problem?

**Comment by Lionel2298** [comment]
What version of GN are you running on ?

**Comment by alexandre433** [comment]
I've had the same issue, for about 3-4 days, it's still at 99.77%, some times i get it to 99.80% and when I turn off the computer and turn it on, the total has changed and it goes back to 99.77% sometimes 99.75%.

I've checked the logs and it has a bunch of `validateSourceAdvanced failed` warnings.

Edit:
There isn't a processing issue i'd say, since I have an i9 10900k and a 3080.

Version of Guardian Node - 2.2.10

**Comment by Lionel2298** [comment]
@alexandre433 Ate you online at the moment ?

**Comment by Lionel2298** [comment]
@alexandre433 Are you online at the moment ?

**Comment by Lionel2298** [comment]
@alexandre433 Has your issue been  resolved ?

**Comment by alexandre433** [comment]
Yes, I deleted the db and reinstalled the program and it worked well.

**Comment by saintscribe69** [comment]
tbh i gave up in the end!!

**Comment by seocrow** [comment]
I have seen this issue on multiple systems including Linux UI Guardian node and Linux console. I see there was a recent update to Windows GN to fix syncing and speed it up. Will this be released for Linux Guardian Nodes also?
